src/utils.rs

```diff
@@ -255,6 +255,6 @@ mod test {
         // A trailing padding byte is incorrect
         let mut v = pad(vec![42], 128).unwrap();
```

Comment:
While we're at it, can you convert *all* instances of `.unwrap()` into `unwrap!`?

Reply:
Done.